src/components/DateRangePicker.jsx

```diff
@@ -43,6 +43,7 @@ const defaultProps = {
   numberOfMonths: 2,
   showClearDates: false,
   disabled: false,
+  reopenPickerOnClear: true,
```

Comment:
This won't work either - that's why i said "default false prop" in my comment.

Absence and `false` should always be the same - so we'll have to pick a name for this prop like "don't reopen picker on clear" (but something much better) and leave the default as `false`.